libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccount.m

```diff
@@ -421,8 +322,8 @@ - (NSString*)description {
     theFullName = [NSString stringWithFormat:@"%@ %@",_idData.firstName,_idData.lastName];
 #endif
     
-    NSString * s = [NSString stringWithFormat:@"<SFUserAccount username=%@ fullName=%@ accessScopes=%@ credentials=%@, community=%@>",
-                    theUserName, theFullName, _accessScopes, _credentials, _communityId];
+    NSString * s = [NSString stringWithFormat:@"<SFUserAccount username=%@ fullName=%@ accessScopes=%@ credentials=%@>",
+                    theUserName, theFullName, _accessScopes, _credentials];
```

Comment:
Community ID is a part of the credentials description so it'll be visible there too, but if anyone wants I can change to `_credentials, _credentials.communityId` instead of deleting